package.json

```diff
@@ -31,5 +32,16 @@
     "stylus-loader": "^3.0.1",
     "webpack": "^2.2.1",
     "xstream": "^10.3.0"
+  },
+  "devDependencies": {
+    "husky": "^0.13.2",
+    "lint-staged": "^3.3.2",
+    "prettier": "^0.22.0"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
+      "git add"
+    ]
```

Comment:
Yes, that's the Prettier way, adopted from gofmt and refmt. The code should be ASTed and re-printed before commit, so that it reflects pure and consistent structure.